uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt5/client/paho/MqttConnectionImpl.java

```diff
@@ -348,12 +354,14 @@ private class MqttMessageListener implements IMqttMessageListener {
         @Override
         public void messageArrived(String topic, MqttMessage mqttMessage) {
             MqttProperties receivedProperties = mqttMessage.getProperties();
+
             String contentType = receivedProperties != null ? receivedProperties.getContentType() : null;
+            Boolean payloadFormatIndicator = receivedProperties != null ? receivedProperties.getPayloadFormat() : null;
```

Comment:
I remember on other line we have another check for receivedProperties != null.

I think it can be joined to single if () with checks for content type and payload format indicator insize.

Reply:
fixed in #22bef55561abf1d1baf3276bcedc8924b9a70f40

Reply:
Please write like here 'fixed in 22bef55561abf1d1baf3276bcedc8924b9a70f40'